helpers/bumper.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python2
+import re
+import os
+import glob
+import subprocess
+import fileinput
+
+os.chdir('../')
```

Comment:
Totally not me: _blindly executes `./helpers/bumper.py`_
Totally not me: ooooh 🤦‍♂️ 

Consider some guardrails to ensure this gets executed at the right level.

Reply:
🤦‍♂️ Yeah I'll make sure that it also goes to the parent directory from where the file is in a separate PR. The chance of the globs actually matching something is pretty low though...but still.

Reply:
Yeah definitely user error, it only touched README's for me across elastic's repositories.